experimental/devices/pca9685/pca9685.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package pca9685
+
+import (
+	"math"
+	"time"
+
+	"periph.io/x/periph/conn/physic"
+
+	"periph.io/x/periph/conn/i2c"
+)
+
+// PCA9685Address i2c default address
+const PCA9685Address uint16 = 0x40
+
+// PCA9685 Commands
+const (
+	Mode1      byte = 0x00
+	Mode2      byte = 0x01
+	SubAdr1    byte = 0x02
+	SubAdr2    byte = 0x03
+	SubAdr3    byte = 0x04
+	Prescale   byte = 0xFE
+	Led0OnL    byte = 0x06
+	Led0OnH    byte = 0x07
+	Led0OffL   byte = 0x08
+	Led0OffH   byte = 0x09
+	AllLedOnL  byte = 0xFA
+	AllLedOnH  byte = 0xFB
+	AllLedOffL byte = 0xFC
+	AllLedOffH byte = 0xFD
+
+	// Bits
+	Restart byte = 0x80
+	Sleep   byte = 0x10
+	AllCall byte = 0x01
+	Invrt   byte = 0x10
+	OutDrv  byte = 0x04
+)
+
+// Dev is a handler to pca9685 controller
+type Dev struct {
+	dev *i2c.Dev
+}
+
+// NewI2CAddress returns a Dev object that communicates over I2C on a alternate address
```

Comment:
Merge these two functions, have the caller specify `pca9685.I2CAddr` every time and comment that it's what should be normally passed.